pkg/operator/base.go

```diff
@@ -46,10 +46,10 @@ func newBaseOperator(
 	return *base
 }
 
-func (b *baseOperator) standardDiff(_ context.Context) map[string]any {
+func (b *baseOperator) operationDiff(_ context.Context) map[string]any {
 	diff := make(map[string]any)
-	diff["before"] = b.resources[beforeDiffField]
-	diff["after"] = b.resources[afterFieldDiff]
+	diff[beforeDiffField] = b.resources[beforeDiffField]
```

Comment:
ocd: `beforeDiffField` and `afterFieldDiff`, dyslexic hehe
not your fault of course, it was already like this. I don't know how I didn't see it before

Reply:
Thank you!!!